message QuerySectionsUplinkPayload {
  required bool queried = 1 [default = true];
}